packages/contracts-bedrock/test/L2/L1Block.t.sol

```diff
@@ -315,3 +315,115 @@ contract L1Block_SetL1BlockValuesIsthmus_Test is L1Block_TestInit {
         assertEq(data, expReturn);
     }
 }
+
+/// @title L1Block_SetL1BlockValuesJovian_Test
+/// @notice Tests the `setL1BlockValuesJovian` function of the `L1Block` contract.
+contract L1Block_SetL1BlockValuesJovian_Test is L1Block_TestInit {
+    /// @notice Tests that setL1BlockValuesJovian updates the values appropriately.
+    function testFuzz_setL1BlockValuesJovian_succeeds(
+        uint32 baseFeeScalar,
+        uint32 blobBaseFeeScalar,
+        uint64 sequenceNumber,
+        uint64 timestamp,
+        uint64 number,
+        uint256 baseFee,
+        uint256 blobBaseFee,
+        bytes32 hash,
+        bytes32 batcherHash,
+        uint32 operatorFeeScalar,
+        uint64 operatorFeeConstant,
+        uint16 daFootprintGasScalar
+    )
+        external
+    {
+        bytes memory functionCallDataPacked = Encoding.encodeSetL1BlockValuesJovian(
+            baseFeeScalar,
+            blobBaseFeeScalar,
+            sequenceNumber,
+            timestamp,
+            number,
+            baseFee,
+            blobBaseFee,
+            hash,
+            batcherHash,
+            operatorFeeScalar,
+            operatorFeeConstant,
+            daFootprintGasScalar
+        );
+
+        vm.prank(depositor);
+        (bool success,) = address(l1Block).call(functionCallDataPacked);
+        assertTrue(success, "Function call failed");
+
+        assertEq(l1Block.baseFeeScalar(), baseFeeScalar);
+        assertEq(l1Block.blobBaseFeeScalar(), blobBaseFeeScalar);
+        assertEq(l1Block.sequenceNumber(), sequenceNumber);
+        assertEq(l1Block.timestamp(), timestamp);
+        assertEq(l1Block.number(), number);
+        assertEq(l1Block.basefee(), baseFee);
+        assertEq(l1Block.blobBaseFee(), blobBaseFee);
+        assertEq(l1Block.hash(), hash);
+        assertEq(l1Block.batcherHash(), batcherHash);
+        assertEq(l1Block.operatorFeeScalar(), operatorFeeScalar);
+        assertEq(l1Block.operatorFeeConstant(), operatorFeeConstant);
+        assertEq(l1Block.daFootprintGasScalar(), daFootprintGasScalar);
+
+        // ensure we didn't accidentally pollute the 128 bits of the sequencenum+scalars slot that
+        // should be empty
+        bytes32 scalarsSlot = vm.load(address(l1Block), bytes32(uint256(3)));
+        bytes32 mask128 = hex"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF00000000000000000000000000000000";
+
+        assertEq(0, scalarsSlot & mask128);
+
+        // ensure we didn't accidentally pollute the 128 bits of the number & timestamp slot that
+        // should be empty
+        bytes32 numberTimestampSlot = vm.load(address(l1Block), bytes32(uint256(0)));
+        assertEq(0, numberTimestampSlot & mask128);
+    }
+
+    /// @notice Tests that `setL1BlockValuesJovian` succeeds if sender address is the depositor
+    function test_setL1BlockValuesJovian_isDepositor_succeeds() external {
```

Comment:
Seems like this test is a weaker version of `testFuzz_setL1BlockValuesJovian_succeeds` and perhaps can be deleted?

Reply:
I agree. Seems this was just following the convention of previous tests, where the same duplication exists. I'd just leave it in, ok?

Reply:
ok, that SGTM, thanks for the attention to detail! :)